### PR TITLE
Feature/added jaeger for local open telemetry

### PR DESCRIPTION
**Pull request by dawiddworak88**
Hi @SebaKas21,

could you please check out this branch, perform the following list of steps and let me know what you see? 

1. Start without debugging in Visual Studio
2. Play with Buyer and Seller Web a little bit
3. Open http://host.docker.internal:3000/d/p34uLaAVk/b2b?orgId=1&refresh=5s (admin -admin) and http://host.docker.internal:16686/search

Cheers,

### CODE REVIEW

File: be/src/DockerServices/otel-collector/otel-collector-config.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    tls:
+      insecure: true
+  prometheusremotewrite:
+    endpoint: "http://prometheus:9090/api/v1/write"
+    tls:
+      insecure: true
+  loki:
+    endpoint: "http://loki:3100/api/prom/push"
+  
+
```
Comment (SebaKas21) [inline be/src/DockerServices/otel-collector/otel-collector-config.yaml:26]: **suggestion:** Redundant new line

**Comment by SebaKas21** [comment]
Hi @dawiddworak88 

Jeager is working, but from grafana i received error "Dashboard not found"

File: be/src/DockerServices/otel-collector/otel-collector-config.yaml (hunk 1 above)
Comment (dawiddworak88) [reply be/src/DockerServices/otel-collector/otel-collector-config.yaml:26]: Fixed

**Comment by dawiddworak88** [comment]
Hi @SebaKas21, could you whether you can see any predefined B2B dashboards after starting and playing with buyer and seller web?

**Comment by dawiddworak88** [comment]
@SebaKas21 : if it was added it should be available in the Dashboards tab from the home page of Grafana

**Comment by SebaKas21** [comment]
@dawiddworak88: Okay, give me few seconds for check :D

**Comment by SebaKas21** [comment]
@dawiddworak88 

In Grafana i don't have predefined dashboards. Only add datasources and "Create your first Dashboard".

Look below

obraz

**Comment by dawiddworak88** [comment]
and if you try to use the menu on the left and find dashboards?

**Comment by SebaKas21** [comment]
Everywhere results not found

obraz

**Review by SebaKas21 — APPROVED** [review]
It works 👍

**Pull request merged**